tests/reprod/test_unsupervised_graphsage.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hmm I'm not sure where else it would fit in, since I think there's currently no "integration"-like tests for an algorithm (i.e. a combination of a pipeline made from components of `layer`, `mapper`, and `data`)?

I did also consider calling this module `eval` instead of `reprod` to keep it more generic